### ISSUE TITLE
Calc: light mode grid vs dark mode with inverted background grid

### ISSUE THREAD

**Issue by ornanovitch** [CollaboraOnline/online#10790]
### Describe the Bug
When in dark mode with invert background toggled, the grid color does not match the light mode grid color

| light | inverted background |
|--------|--------|
| ![Screenshot 2024-12-21 at 14-33-37 blabla ods - Brume](https://github.com/user-attachments/assets/25e902b8-c30b-4abf-98f4-4d100b271a14)  |  ![Screenshot 2024-12-21 at 14-33-19 blabla ods - Brume](https://github.com/user-attachments/assets/3693a75c-36a2-4fce-8693-3bacd82feaed)  | 

### Expected Behavior
The grid color with inverted background in dark mode should match the grid color in light mode.

On CODE 24.04.11.1

**Comment by Ornanovitch** [comment]
Could it be something missing from https://github.com/CollaboraOnline/online/pull/10575?

**Comment by Ezinnem** [comment]
@pedropintosilva please can you take a look?

**Comment by jazevedo-coll** [comment]
> Could it be something missing from https://github.com/CollaboraOnline/online/pull/10575

@Ornanovitch yes, a bisect shows this is from: https://github.com/CollaboraOnline/online/pull/10575

**Comment by smehrbrodt** [comment]
Fixed with https://gerrit.collaboraoffice.com/plugins/gitiles/online/+/4b5830336d60db23f894bda432c013b14b22e3e8%5E%21/

**Comment by pedropintosilva** [comment]
Thanks @smehrbrodt !
